### ISSUE TITLE
Support OpenAPI schema version without revision

### ISSUE THREAD

**Issue by nielsvanvelzen** [jellyfin/jellyfin-sdk-kotlin#206]
An upcoming server update (probably 10.8) will change the format from `Major.Minor.Build.Revision` to `Major.Minor.Build` (10.7.0.0 -> 10.7.0). We need to validate if the generator and core library keep behaving as we expect. It's _probably_ fine though.

Reference: jellyfin/jellyfin#5661

**Comment by nielsvanvelzen** [comment]
Tested this by changing the version properties in the OpenAPI doc and generating the sources. The apiVersion in ApiConstants changed (as expected). I looked trough all usages for the property and it's only used in the Jellyfin class where it's converted to a `ServerVersion` instance. This data class doesn't require the revision (it's null by default). So I don't expect any issues with this change.

TL;DR: I don't expect any issues with jellyfin/jellyfin#5661 so backporting should be fine.
